### ISSUE TITLE
Enable associating multiple islands with a given URL fragment

### ISSUE THREAD

**Issue by nathanshelly** [natasha-codes/firefox-islands#7]

**Comment by sashaweiss** [comment]
per offline conversation, probably want some kind of basic picker saying "hey you have multiple islands this could go to, which do you want?".

**Comment by sashaweiss** [comment]
thought here: since we want to spruce up the options page (ik you @nathanshelly were excited about svelte?) we could use the same for whatever popup or something that we want to show in response to this.

**Comment by nathanshelly** [comment]
Yeah was thinking about Svelte for this context originally (could also work on the options page). I have since thought it might be overkill for the moment (though would be fun to try at some point).
